persist/sqlite/consensus.go

```diff
@@ -1059,10 +1079,9 @@ func revertSpentSiafundElements(tx *txn, elements []types.SiafundElement) error
 			balanceChanges[addrRef.ID] = addrRef.Balance
 		}
 
-		var dummy types.Hash256
-		if err := stmt.QueryRow(encode(se.ID)).Scan(decode(&dummy)); err != nil && !errors.Is(err, sql.ErrNoRows) {
+		if res, err := stmt.Exec(encode(se.ID)); err != nil {
 			return err
-		} else if errors.Is(err, sql.ErrNoRows) {
+		} else if n, _ := res.RowsAffected(); n == 0 {
```

Comment:
What made you change this?

Reply:
Why scan a dummy row when SQLite will return the count of updated rows?

Reply:
Indeed but that reason felt unrelated enough to this PR that I thought maybe the previous approach didn't work and that's why you changed it.